### DISCUSSION TITLE
Use HTML in q-input suffix

### DISCUSSION THREAD

**Discussion by allard-potma** [quasarframework/quasar#8876] [General - Components / Directives / etc] [answered]
I want to make a part of my suffix on a q-input in italic. I succeeded to do this in my q-select using :options-html, is there a similar thing for a q-input suffix?
![unit](https://user-images.githubusercontent.com/59834634/114531220-fe6d7a80-9c4b-11eb-9385-e66db506fdcf.png)

**Comment by dvbeek** [comment, marked as answer]
I think you can use the append slot for this.

I.e.:
```
      <q-input model-value="test" title="test">
        <template v-slot:append>
          <span class="text-grey-9 text-body2"><small>gram <i>(g)</i></small></span>
        </template>
      </q-input>
```
